.travis.yml

```diff
@@ -44,3 +45,6 @@ install:
 
 script:
 - testing/run_test_suite.sh
+
+after_script:
```

Comment:
I think it'd be better to make the script name `test_cleanup.sh` or something along those lines, to match with the generic `install_test_dependencies.sh`. That way, if we have additional cleanup tasks/scripts, we can add them there.